visallo/visallo
# Remove dependency on SimpleORM

Testing Instructions:
- Verify vertex thumbnails still work
- Verify video thumbnails still work
- Verify user/system notifications still work

CHANGELOG
Added: 
Changed: 
Documentation: 
Deprecated:
Removed: dependency on SimpleORM
Fixed:
Security:
